libs/blocks/merch-card/merch-card.js

```diff
@@ -102,6 +102,14 @@ function addMerchCardGridsIfMissing(section) {
   }
 }
 
+const decorateMerchCardLinkAnalytics = (el) => {
+  [...el.querySelectorAll('a')].forEach((link, index) => {
+    const heading = el.querySelector('h3');
+    const analyticsString = `${processTrackingLabels(link.textContent)}-${index + 1}|${heading.textContent}`;
+    link.setAttribute('daa-ll', analyticsString);
```

Comment:
Is it certain that links will always contain this `h3`? Otherwise, a check might be beneficial, since `heading.textContent` is referenced